KomaMRIBase/src/KomaMRIBase.jl

```diff
@@ -0,0 +1,64 @@
+module KomaMRIBase
+
+#IMPORT PACKAGES
+import Base.*, Base.+, Base.-, Base./, Base.vcat, Base.size, Base.abs, Base.getproperty
+#General
+using Reexport
+#Datatypes
+using Parameters
+#Simulation
+using Interpolations
+#Reconstruction
+using MRIBase
+@reexport using MRIBase: Profile, RawAcquisitionData, AcquisitionData, AcquisitionHeader, EncodingCounters
+using MAT   # For loading example phantoms
+
+global γ = 42.5774688e6; # Hz/T gyromagnetic constant for H1, JEMRIS uses 42.5756 MHz/T
+
+# Hardware
+include("datatypes/Scanner.jl")
+# Sequence
+include("datatypes/sequence/Grad.jl")
+include("datatypes/sequence/RF.jl")
+include("datatypes/sequence/ADC.jl")
+include("timing/KeyValuesCalculation.jl")
+include("datatypes/Sequence.jl")
+include("datatypes/sequence/Delay.jl")
+# Phantom
+include("datatypes/Phantom.jl")
+# Simulator
+include("datatypes/simulation/DiscreteSequence.jl")
+include("datatypes/simulation/Spinor.jl")
+include("datatypes/simulation/Magnetization.jl")
+include("datatypes/simulation/DiffusionModel.jl")
+include("timing/TimeStepCalculation.jl")
+include("timing/TrapezoidalIntegration.jl")
+
+# Main
+export γ    # gyro-magnetic ratio [Hz/T]
+export Scanner, Sequence, Phantom
+export Grad, RF, ADC, Delay
+export Mag, dur
+export DiscreteSequence
+export SpinStateRepresentation
+export discretize, get_sim_ranges, get_adc_phase_compensation, kfoldperm, trapz, cumtrapz, mul!, get_adc_sampling_times
```

Comment:
only preserve `discretize`, `get_adc_phase_compensation`, `get_adc_sampling_times`

Reply:
Done. Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/8cf10212c4249fd8944bf303f0330a03dd071cca